xcode/Shared (App)/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,7 @@
+/* 
```

Comment:
is this file needed?

Reply:
Yes, without that file, the translations with the export function generate duplicates for both projects (macOS, iOS). After the translation to other languages, this is a source file for the different versions.